### PR TITLE
Feature/suggest games

### LINKED ISSUE #57: As a gamer owner, I want others to able to suggest games, so that I know what new games others want to play
- [ ] Store requested games
- [ ] Show requested games on the platform 
- [ ] Suggest new games

### PR DESCRIPTION
**Pull request by EmilMadsstuen**
You can now suggest games, fixes #57

### CODE REVIEW

**Review by x183 — APPROVED** [review]
Seems to work just fine :+1:

File: backend/src/routers/suggestRouter.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+		if (body.releaseDate > new Date().toISOString()) {
+			return sendApiValidationError(
+				res,
+				{
+					path: 'releaseDate',
+					message: 'The release date cannot be in the future'
+				},
+				'Body'
+			);
+		}
```
Comment (Supergamer1337) [inline backend/src/routers/suggestRouter.ts:104]: I think, like your own comment says, this should be the case. Especially for suggested games, as those are even more likely to be later releases than games that someone currently has.
```suggestion
```

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
Except for the suggestion below, I have a question. Wouldn't it be better to remove some of the options to store? For a suggestion I think actually having all this information is in some ways superflous? I don't really know if I need the expected playtime for example.

Then again, I guess it does make it easeier for the person buying the games if they have that information up front rather than having to look for it themselves?

**Comment by EmilMadsstuen** [comment]
> Except for the suggestion below, I have a question. Wouldn't it be better to remove some of the options to store? For a suggestion I think actually having all this information is in some ways superflous? I don't really know if I need the expected playtime for example.
> 
> Then again, I guess it does make it easeier for the person buying the games if they have that information up front rather than having to look for it themselves?

We had the same thoughts when we wrote the code. In some ways, it feels unnecessary to include the extra data, however as an organization, I would probably want to know some of the extra information like platform (do we have that platform?) and Min/Max player so I know what to prioritize buying (those with low min and high max count). We are fine with changing if you think its better to include as little info as possible.

**Comment by Supergamer1337** [comment]
I will accept it, but I just think it's good to have in mind. Personally, I think that having too much information right now, that could easily be removed if our shareholder want us to is better than having to rewrite it.

**Review by Supergamer1337 — APPROVED** [review]

**Pull request merged**